controls/datagrid/columns/header.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Please check the [first heading](https://github.com/telerik/maui-docs/blob/a24d52c04cadfe514bf36d5259864bd5ba3e5a80/controls/datagrid/columns/header.md#changing-the-text-in-the-header) in the article and edit them by implementing [title-case capitalization](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).